### PR TITLE
made my signup page

### PR DESCRIPTION
**Pull request by GarvG**
Please Review My PR Created my Sign-up Page

### CODE REVIEW

File: Sign-up.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <div class="col-2">
+                    <img src="IMAGES/home.jpg" class="Img-login">
+                </div>
+                <div class="col-2 col-form">
+                    <div class="form-container register-form">
+                        <div class="form-btn">
+                            <span class="Register-content">Not Registered Yet!</span>
+                        </div>
+                        <form>
+                            <input type="text" placeholder="UserName" max="10" min="4" required>
```
Comment (itsmebipul00) [inline Sign-up.html:51]: use label tags

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Good to merge with changes

**Review by archit7448 — APPROVED** [review]
good to merge

File: Sign-up.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| Login-Signup</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="login.css">
+</head>
+
+<body>
+    <div class="header">
```
Comment (purvasharmadev) [inline Sign-up.html:14]: You can use header tag here.

**Review by purvasharmadev — APPROVED** [review]
Try using more semantic HTML, else everything looks good to go.

File: Sign-up.html (hunk 1 above)
Comment (GarvG) [reply Sign-up.html:51]: did not used intentionally want the plain look

**Pull request merged**